### DISCUSSION TITLE
Blueprint Runtime Error. -UEReferenceApp

### DISCUSSION THREAD

**Discussion by Yuhandswash** [JSBSim-Team/jsbsim#841] [Q&A]
I'm trying to run UEReferenceApp according to the tutorial provided, but my plane could not take off.
 I pressed the number 4 and the plane did not take off, only some commands worked, such as pressing the "TAB" key and "G" key and commands to change the time of day. An error occurs when I press esc to exit.

![WZZ HTP0U$C8ASTD8P(1 _1](https://user-images.githubusercontent.com/77879596/220905920-29a66596-7546-47f5-808a-c856f0c9e8e2.png)
![}TXPPXLEJ$)WX(LG32L6DOR](https://user-images.githubusercontent.com/77879596/220905942-4a7655ba-9a63-4312-bfb9-7350fd4a0c1f.png)

**Comment by Yuhandswash** [comment]
I'm using Unreal Engine 5.0.3, Win10 system.One more thing: Blueprints compile successfully.I'm confused and don't know how to fix it.Thanks

**Comment by gallonmate** [comment]
I'm assuming you have changed the aircraft model. The Unreal Engine plugin is a developer's starting point and many features do not exist.
The Error you're seeing is just saying that Unreal cannot find any aircraft engines. Because not all aircraft engines are supported by default. For example the X15 rocket aircraft uses rocket engines, which are not supported by default. The entire plugin is just an example/starting point. You can edit your blueprint (or edit the plugin in c++) to remove these errors but you'll still have to figure out how to start the rocket engine.

**Reply by Yuhandswash** [reply]
Thank you very much for your answer
